qdrant/v1.2.x/search.md

```diff
@@ -592,3 +592,176 @@ It is impossible to retrieve Nth closest vector without retrieving the first N v
 However, using the offset parameter saves the resources by reducing network traffic and the number of times the storage is accessed.
 
 Using an `offset` parameter, will require to internally retrieve `offset + limit` points, but only access payload and vector from the storage those points which are going to be actually returned.
+
+## Grouping API
+
+*Available since v1.2.0*
+
+It is possible to group results by a certain field. This is useful when you have multiple points for the same item, and you want to avoid redundancy of the same item in the results.
+
+For example, if you have a large document split into multiple chunks, and you want to search or recommend on a per-document basis, you can group the results by the document ID. 
+
+Consider having points with the following payloads:
+
+```json
+{
```

Comment:
Maybe we should add some dummy vectors, just with one dimension, to at least make the JSON object valid.